README.rst

```diff
@@ -698,7 +698,7 @@ caught programmatically:
 Specification
 =============
 
-See :ref:`specification`.
+See https://morphio.readthedocs.io/en/latest/specification.html
```

Comment:
pypi does not see this link in another page, so it fails. I will fix properly in #277